src/components/Planning/PlanningTable/PlanningRow/PlanningHead.tsx

```diff
@@ -42,8 +78,32 @@ const PlanningHead: FC<IPlanningHeadProps> = ({
             {angleIcon}
           </button>
           {type !== 'division' && (
-            <div className={`planning-head-content-dots`} data-testid={`show-more-${id}`}>
-              <IconMenuDots size="s" />
+            <div
+              className={`planning-head-content-dots cursor-pointer`}
+              data-testid={`show-more-${id}`}
+            >
+              {type === 'group' && (
+                <>
+                  <GroupDialog
+                    isOpen={groupDialogState.groupEditOpen}
+                    handleClose={onCloseGroupEditDialog}
+                    id={id}
+                    editMode={true}
+                    projects={projectsToIOption()}
+                  />
+                  <DeleteGroupDialog
+                    isVisible={groupDialogState.groupDeleteOpen}
+                    onCloseDeleteGroupDialog={onCloseGroupDeleteDialog}
+                    groupName={name}
+                    id={id}
+                  />
+                </>
+              )}
+              <IconMenuDots
+                size="s"
+                onMouseDown={type === 'group' ? handleGroupRowMenu : undefined}
```

Comment:
If keeping mouse down constantly this will invoke handleGroupRowMenu function all the time. Use onClick instead.